spec/features/friend_invitation_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'spec_helper'
 
+Capybara.register_driver :selenium do |app|
+    Capybara::Selenium::Driver.new(app, :browser => :chrome)
+end
```

Comment:
Changing the driver like that will not work that easily. You have to change RSpec settings and add DatabaseCleaner set up. Also you should label the spec as `js: true`. You will face a lot of feature specs that will test javascript in your apps - its better to have this setup in `spec_helper.rb` or `rails_helper.rb`